parsers/powershell.c

```diff
@@ -72,8 +71,24 @@ typedef enum eTokenType {
 	TOKEN_VARIABLE
 } tokenType;
 
+enum {
+	KEYWORD_function,
+	KEYWORD_filter,
+	KEYWORD_class,
+};
+
+/* We need an integer that is not an unsigned to allow KEYWORD_NONE. */
+typedef int keywordId;
+
+static const keywordTable PowerShellKeywordTable[] = {
+	{ "function",	KEYWORD_function },
+	{ "filter",		KEYWORD_filter },
```

Comment:
If you choose "unifying-function-and-filter approach", this line should be:
```
+              { "filter",		KEYWORD_function },
```

Reply:
We chose the "introducing-a-kind-for-filter approach". For more information, see https://github.com/universal-ctags/ctags/pull/3480#discussion_r977902309.